### ISSUE TITLE
When using apt to install the application from Cuba, the repository respond with 403 error.

### ISSUE THREAD

**Issue by reyespinosa1996** [shiftkey/desktop#657]
### Operating System/Distribution

Which distribution of Linux are you using?
Debian Stable (v11 Bullseye) amd64

### Installer

Which version of the app?
The latest version available right now.
Which installer type?
.deb amd64
### What happened?

```
Err:6 https://packagecloud.io/shiftkey/desktop/any any InRelease               
  403  Forbidden [IP: 192.168.1.1 3128]
Reading package lists... Done                          
E: Failed to fetch https://packagecloud.io/shiftkey/desktop/any/dists/any/InRelease  403  Forbidden [IP: 192.168.1.1 3128]
E: The repository 'https://packagecloud.io/shiftkey/desktop/any any InRelease' is not signed.

```
Probably this issue is not related with a misconfiguration of the repository but due to the trade embargo imposed by the US government to Cuban citizens. I'm asking politely to look if there is any change to make a mirror available with another service (Maybe something like the Suse Open Build System will work, I know it works here in Cuba without 403 error)

**Comment by mwt** [comment]
Are you able to access my mirror? Though, it's in US. #647

The IP in the error is an internal IP address for some reason.

**Comment by reyespinosa1996** [comment]
Yes, because is my proxy answering the client machine, but is reflecting the response from the server.

**Comment by reyespinosa1996** [comment]
> Are you able to access my mirror? Though, it's in US. #647
> 
> The IP in the error is an internal IP address for some reason.

Could you give me the address of your mirror? If it's not in Cloudfare or Sourceforce must of the time it works.

**Comment by mwt** [comment]
> Could you give me the address of your mirror? If it's not in Cloudfare or Sourceforce must of the time it works.

The url is <https://mirror.mwt.me/ghd/deb/>. There are more instructions in the issue number above if you need them. That link isn't through Cloudflare/Sourceforce.

**Edit:** you can use <https://mirror.mwt.me/ghd/gpgkey> to get the gpgkey if (and only if) you weren't able to get that from packagecloud either.

**Comment by reyespinosa1996** [comment]
Thank you, @mwt this is really helpful.

**Comment by mwt** [comment]
> Thank you, @mwt this is really helpful.

Awesome! I'm glad it's working well for you.

**Comment by shiftkey** [comment]
@reyespinosa1996 thanks for the report and thanks @mwt for the assistance!

Unfortunately the package feed is hosted on PackageCloud, and I have limited control over this so I'm going to close this out now that we've resolved this.

@mwt if you're interested in becoming a more prominent mirror I'd happily accept some changes to the `README` in this repository, or I can take some notes from https://github.com/shiftkey/desktop/issues/647 whenever you're feeling the setup instructions are good to go.
